src/app/components/Header/index.tsx

```diff
@@ -1,21 +1,29 @@
 'use client'
+import { MENU, SITE_NAME } from '@/app/constants'
 import { Container, Nav, Navbar, Offcanvas } from 'react-bootstrap'
-import { SITE_NAME } from '@/app/constants'
 
 export function Header() {
   return (
     <header>
       <Navbar expand="lg" className="bg-body-tertiary">
         <Container>
-          <Navbar.Brand href="/">{SITE_NAME}</Navbar.Brand>
+          <Navbar.Brand href="/">
+            <h1>{SITE_NAME}</h1>
+          </Navbar.Brand>
           <Navbar.Toggle aria-controls="main-navbar-nav" />
           <Navbar.Offcanvas id="main-navbar-nav" placement="end">
             <Offcanvas.Header closeButton>
               <Offcanvas.Title>{SITE_NAME}</Offcanvas.Title>
             </Offcanvas.Header>
             <Offcanvas.Body>
-              <Nav className="justify-content-end flex-grow-1 pe-3">
-                <Nav.Link href="/">Home</Nav.Link>
+              <Nav className="justify-content-start flex-grow-1 pe-3">
+                {MENU.map(function (menu) {
+                  return (
+                    <Nav.Link className="mx-5" href="/">
+                      {menu.name}
+                    </Nav.Link>
+                  )
+                })}
```

Comment:
Please pay attention to console errors. There should be no need to tell you these basic things anymore. If you are mapping a list you **must** add `key` prop!
```suggestion
                {MENU.map ((menu) => (
                    <Nav.Link className="mx-5" href={menu.link} key={menu.name}>
                      {menu.name}
                    </Nav.Link>
                ))}
```

Reply:
done